tfhe/Cargo.toml

```diff
@@ -64,6 +64,11 @@ shortint = []
 integer = ["shortint"]
 internal-keycache = ["lazy_static", "fs2", "bincode"]
 
+# Experimental section
+experimental = ["experimental-multi_bit_pbs"]
+experimental-multi_bit_pbs = []
```

Comment:
I find it a bit weird to mix `-` and `_` in the same feature name

Reply:
Can be all dashes if you think it's better

Reply:
Its fine to leave it like that i think, it should be temporary ™

Reply:
It was mainly to mark the fact that this is experimental and the underlying feature is multi_bit_pbs